### ISSUE TITLE
Keyboard focus will get invisible after collapse configure notification button with the help of Esc key.

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/vscode#108296]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#Win32;#DesktopApp;#A11y_VSCode;#WCAG2.4.3;#A11ySev2;#CAITestPass_May2020;#Benchmark;#Visual Studio Code;

**Environment Details:**
Application Name: Azure Extensions for VS Code
Version: 1.46.1 (user setup)
Commit: cd9ea6488829f560dc949a8b2fb789f3cdc05f5d
Date: 2020-06-17T21:13:20.174Z
Electron: 7.3.1
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363
Windows Version: Windows10

**Repro Steps:**

1. 1. Open Visual studio code
2. 2. Click on extension button and search for "Azure Functions"
3. 3. Click on install and move to the "Azure" button
4. 4. Click on"Local Project" from the treeview and expand it.
5. 5. Click on the Functions menu and expand it further to see Triggers.
6. 6. Right click on trigger to see the menu item "Disable Function"
7. Navigate to configure notification button and expand and then collapse with the help of esc key.

**Actual:**

Keyboard focus will get invisible after collapse configure notification button with the help of Esc key.

**Same issue is also observed with throughout the application.**

**Expected:**

Focus should be remain constant on the button from where we have started to perform action. Focus should move to "Configure notification button.

**User Impact:**

User will get confuse because of illogical focus order.

**Attachment:**
[#108296.zip](https://github.com/microsoft/vscode/files/5347256/108296.zip)

**Comment by aeschli** [comment]
As seen in the attached video this is about the `configure` button in the notification.

**Comment by isidorn** [comment]
This works fine for me.
Please use the Notifications Tester extension and latest vscode insiders to file an issue with a good minimal description. Thanks

**Comment by isidorn** [comment]
GitHubTags:#Closed;
